server/src/com/thoughtworks/go/server/controller/ArtifactsController.java

```diff
@@ -239,28 +236,6 @@ public ModelAndView putArtifact(@RequestParam("pipelineName") String pipelineNam
 
     /* Other URLs */
 
-    @RequestMapping(value = "/**/consoleout.json", method = RequestMethod.GET)
-    public ModelAndView consoleout(@RequestParam("pipelineName") String pipelineName,
-                                   @RequestParam("pipelineLabel") String counterOrLabel,
-                                   @RequestParam("stageName") String stageName,
-                                   @RequestParam("buildName") String buildName,
-                                   @RequestParam(value = "stageCounter", required = false) String stageCounter,
-                                   @RequestParam(value = "startLineNumber", required = false) Integer start
```

Comment:
I think it's just an internal redirect from [here](https://github.com/gocd/gocd/blob/5478a5460cd4fcb1a7380eb0e4ab3fbbeacb4166/server/webapp/WEB-INF/urlrewrite.xml#L202). So, it's past the acegi bit.